feature_engine/feature_selection.py

```diff
@@ -88,3 +89,138 @@ def transform(self, X):
         return X
 
 
+class SelectFeatures(TransformerMixin, BaseEstimator):
+    """
```

Comment:
Wonder if we should just call the transformer DropConstantFeatures() it is too long otherwise?

Reply:
I was thinking ```DropQuasiConstantFeatures()``` since it gives the user a hint of what it actually does. The threshold of 1 will become constant features. If not 1, it becomes quasi constant + constant. Let me know...

Reply:
I am not sure what is best. Let's go with the long name then: DropConstantAndQuasiConstantFeatures()